Resources/config/default_settings.yml

```diff
@@ -13,3 +13,26 @@ parameters:
     # Type ('js' or 'template')
     # ez_platformui.<scope>.yui.modules.<module_name>.type
     ez_platformui.default.css.files: []
+
+    ezsettings.default.system_info_view:
```

Comment:
oookay. Works here, but if `ezsettings.global.system_info_view` is better we'll go for that :)